.github/workflows/snapshot-build.yml

```diff
@@ -80,6 +80,12 @@ jobs:
             SUBSTRATE_CLI_GIT_COMMIT_HASH=${{ github.sha }}
             RUSTFLAGS=${{ matrix.platform.rustflags }}
 
+      - name: Trigger sec-scan Workflow
+        uses: peter-evans/repository-dispatch@a4a90276d01e3a2ae44fa10a0247287f045afd59 # @v2.1.2
+        with:
+          event-type: sec-scan-trigger
+          client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}" , "tag": "${{ steps.meta.outputs.tags }}" }'
```

Comment:
We only seem to use `tag` on the receiving side, why sending so much stuff?

Also `tags` seems to imply multiple values, but you're using it as a single value instead.

Reply:
changed this to just send that sha hash and use that instead.